### ISSUE TITLE
Cannot bridge two nodes

### ISSUE THREAD

**Issue by bmcustodio** [emqtt/emqttd#206]
I tried to bridge two nodes but I get the following error:

```
bmcustodio :: ~/emqttd ❯ bin/emqttd_ctl bridges start "emqttd@n1.dev.local" "#"
bin/emqttd_ctl: 226: bin/emqttd_ctl: [[: not found
error: {{undef,[{emqttd_mqueue,mqueue,[],[]},
                {emqttd_bridge,init,1,
                               [{file,"src/emqttd_bridge.erl"},{line,50}]},
                {gen_server,init_it,6,[{file,"gen_server.erl"},{line,328}]},
                {proc_lib,init_p_do_apply,3,
                          [{file,"proc_lib.erl"},{line,239}]}]},
        {child,undefined,
               {bridge,'emqttd@n1.dev.local',<<"#">>},
               {emqttd_bridge,start_link,
                              ['emqttd@n1.dev.local',<<"#">>,
                               [{max_queue_len,10000},
                                {ping_down_interval,1}]]},
               transient,10000,worker,
               [emqttd_bridge]}}
```

Am I doing something wrong? I could cluster the two nodes (they do see each other) but cannot bridge them for some reason. I'm using `0.9.1`.

**Comment by erylee** [comment]
@bmcustodio, fixed in 0.9.3 and update Wiki: https://github.com/emqtt/emqttd/wiki/Bridge.

Any suggestions on bridge design will be welcome:)

**Comment by erylee** [comment]
0.9.3 released, and the issue fixed.
